stripe/src/test/java/com/stripe/android/model/CardTest.java

```diff
@@ -526,6 +526,7 @@ public void getLast4_whenNumberIsNullButLast4IsSet_returnsCorrectValue() {
                 "1234",
```

Comment:
Can we add a thing to this test that checks that the `funding` key in a JSON response blob is properly deserialized into the `Card`?